templates/skeleton/app/routes/cart.tsx

```diff
@@ -93,7 +93,7 @@ export default function Cart() {
           errorElement={<div>An error occurred</div>}
         >
           {(cart) => {
-            return <CartMain layout="page" cart={cart} />;
+            return <CartMain layout="page" cart={cart!} />;
```

Comment:
Maybe we should make `CartMain` get `cart: Cart | null` instead of adding `!` everywhere?